test/unified_format.py

```diff
@@ -173,28 +204,28 @@ def __init__(self, test_class):
         self._entities = {}
         self._listeners = {}
         self._session_lsids = {}
-        self._test_class = test_class
+        self.test = test_class
```

Comment:
I renamed this from `_test_class` to `test` to save horizontal space and avoid needing to break up as many lines.